test/unit/package_test.rb

```diff
@@ -233,158 +233,158 @@ def test_centimetres_girth_cylinder
     assert_nil @package.centimetres(:unknown)
   end
 
-  def test_centimetres_alias_cm
+  test "#centimetres alias to cm" do
     assert_equal @dimensions, @package.centimetres
     assert_equal @dimensions, @package.cm
   end
 
-  def test_weight
+  test "#weight" do
     assert_equal @mass, @package.weight
     assert_instance_of Measured::Weight, @package.weight
   end
 
-  def test_weight_actual
+  test "#weight actual" do
     assert_equal @mass, @package.weight(type: :actual)
     assert_instance_of Measured::Weight, @package.weight(type: :actual)
   end
 
-  def test_weight_volumetric
+  test "#weight volumetric" do
     assert_equal Measured::Weight.new(35, :grams), @package.weight(type: :volumetric)
   end
 
-  def test_weight_dimensional
+  test "#weight dimensional" do
     assert_equal Measured::Weight.new(35, :grams), @package.weight(type: :dimensional)
   end
 
-  def test_weight_billable_max_weight_and_volumetric
+  test "#weight billable is the max of weight and volumetric" do
     assert_equal Measured::Weight.new(100, :grams), @package.weight(type: :billable)
 
     @package = Package.new(500, [1, 1, 1], @options)
     assert_equal Measured::Weight.new(500, :grams), @package.weight(type: :billable)
   end
 
-  def test_grams_value
+  test "#grams value" do
     assert_equal 100, @package.grams
   end
 
-  def test_grams_accepts_options_with_type
+  test "#grams accepts options with type" do
     assert_in_delta 35, @package.grams(type: :volumetric), 1
   end
 
-  def test_grams_converts
+  test "#grams converts to another unit" do
     @package = Package.new(@weight, @dimensions, weight_units: :imperial)
 
     assert_in_delta 2834.9, @package.grams, 1
   end
 
-  def test_grams_alias_g
+  test "#grams alias to g" do
     assert_equal @package.grams, @package.g
   end
 
-  def test_ounces_value
+  test "#ounces value" do
     assert_equal 100, @imperial_package.ounces
   end
 
-  def test_ounces_accepts_options_with_type
+  test "#ounces accepts options with type" do
     assert_in_delta 20.2, @imperial_package.ounces(type: :volumetric), 1
   end
 
-  def test_ounces_converts
+  test "#ounces converts to another unit" do
     assert_in_delta 3.5, @package.ounces, 1
   end
 
-  def test_ounces_alias_oz
+  test "#ounces alias to oz" do
     assert_equal @imperial_package.ounces, @imperial_package.oz
   end
 
-  def test_pounds_value
+  test "#pounds value" do
     assert_equal 6.25, @imperial_package.pounds
   end
 
-  def test_pounds_accepts_options_with_type
+  test "#pounds accepts options with type" do
     assert_in_delta 0.07, @package.pounds(type: :volumetric), 0.01
   end
 
-  def test_pounds_converts
+  test "#pounds converts to another unit" do
     assert_in_delta 0.22, @package.pounds, 0.01
   end
 
-  def test_pounds_alias_lb
+  test "#pounds alias to lb" do
     assert_equal @package.pounds, @package.lb
   end
 
-  def test_pounds_alias_lbs
+  test "#pounds alias to lbs" do
```

Comment:
.. and here, and few other spots in the file. Just a suggestion though, if you think it makes sense.